test/users/bguarraci/for_byte.chpl

```diff
@@ -0,0 +1,3 @@
+for i in 254:uint(8)..255:uint(8) {
```

Comment:
@briangu 

Just FYI, it's unlikely that this will ever be supported.

We implement range iteration in terms of "C for-loops" for performance reasons and you can't cleanly loop from 254 to 255 using a "C for-loop". This will roughly get translated into:

```c
for (i=254; i<=255; i+=1) {}
```
and since all uint(8)'s are <= 255 you'd have an infinite loop which is why we warn you.

You could either write this as: 

```chapel
for i in 254:uint(16)..255:uint(16) {
  writeln(i:uint(8));
}
```

or 

```chapel
for i in (254:uint(8)..255:uint(8)).generalIterator() {
  writeln(i);
}
```

note that the general iterator will get terrible performance because it is effectively generated as

```c
i = 254
for (i=254; true; i+=1) {
  writeln(i);
  if (i == 255) { break; }
}
```